storybook-docs/stories/1 - Getting Started.mdx

```diff
@@ -0,0 +1,47 @@
+# Getting Started
```

Comment:
Same as the previous comments as `README.md`

Reply:
applied changes